README.md

```diff
@@ -17,13 +17,20 @@ export function render () {
     { text: 'Tab 2', content: <div>Tab 2 content</div> },
     { text: 'Tab 3', content: <div>Tab 3 content</div> }
   ];
+  // optional onChange event
+  const onChange = ({ clickedIndex }) => {
+    console.log(`you clicked index ${clickedIndex}`)
+  };
 
-  return <TabControl active-tab-index={0} items={items} />;
+  return <TabControl activeTabIndex={0} items={items} onChange={onChange} removeOnHide={true} />;
 }
 
 ```
 
 ### Attributes
 
-#### active-tab-index
+#### activeTabIndex
 Set initial active tab index
+
+### removeOnHide
+If set to true, only the active tab content will be rendered.
```

Comment:
@iefserge why? what would be the benefit of that? or maybe what's the downside of having this option here? If we're going to show/hide the content in another component then this component would be obsolete for our use case imo.